### PR TITLE
Lambdify Jupyter Notebook

### LINKED ISSUE #330: Jupyter notebook example for lambdify in symbolic module
🐞 **Problem**

  - **The title is self-descriptive**

📋  **Steps to solve the problem**

 * Comment below about what you've started working on.
 * Add, commit, push your changes
 * Submit a pull request and add this in comments - `Addresses #<put issue number here>`
 * Ask for a review in comments section of pull request
 * Celebrate your contribution to this project 🎉

### PR DESCRIPTION
**Pull request by OVSofia**
Fixes #330

### CODE REVIEW

**Comment by shreyasbapat** [comment]
Make sure you add all the new notebooks in `index.ipynb` with an image

**Comment by OVSofia** [comment]
> Make sure you add all the new notebooks in `index.ipynb` with an image

Effectively. It had not been saved. I have already fixed it

**Comment by OVSofia** [comment]
I don't know why, but the preview doesn't look like this:

image

**Comment by ritzvik** [comment]
@OVSofia you do not need to make me co-author in the commits in which I didn't contribute!

**Comment by OVSofia** [comment]
> @OVSofia you do not need to make me co-author in the commits in which I didn't contribute!

I was just trying how to do this. Anyway you contributed in a lot of RP and I didn't made you co-author xD

**Comment by ritzvik** [comment]
@OVSofia , I'm suggesting a few changes

  - Firstly, kindly don't use schwarzschild metric here, people would get seriously bored 🤣 . Use any generic tensor. Something like this : 
    ```
    syms = symbols("x y")
    x, y = syms
    T = BaseRelativityTensor([[x, 1],[0, x+y]], syms, config="l")
    ```
  - Secondly, understand that the `tensor_lambdify` returns a two things, a list of arguments and a function. You would have to show how to use that function
    ```
    args, func = T.tensor_lambdify()
    print(args)
    ```
    ```
    print(func(1,2))
    ```

of course, you are free to use your own examples!

**Comment by ritzvik** [comment]
> I don't know why, but the preview doesn't look like this:

After these, we would see about `index.ipynb`

**Comment by ritzvik** [comment]
> > @OVSofia you do not need to make me co-author in the commits in which I didn't contribute!
> 
> I was just trying how to do this. Anyway you contributed in a lot of RP and I didn't make you co-author xD

BTW, you did the commit wrong :) . It's done like this
```
git commit -m "Your commit message

Co-authored-by: Ritwik Saha <s.ritwik98@gmail.com>"
```

Notice the newline between the commit message and co-author. Also, the email used should be the same which is registered by the person on GitHub. Then you would see that GitHub automatically recognizes the author!

**Comment by ritzvik** [comment]
@OVSofia I did a few changes. Can you see the way I have committed, and how GitHub recognizes that ?

File: index.ipynb [hunk 1]
```diff
@@ -61,7 +61,12 @@
     "\n",
     "## [Eisntein Tensor calculations using Symbolic module](docs/source/examples/Einstein_Tensor_symbolic_calculation.ipynb)\n",
     "\n",
-    "[![symbol](docs/source/examples/einstein.png)](docs/source/examples/Einstein_Tensor_symbolic_calculation.ipynb)"
+    "[![symbol](docs/source/examples/einstein.png)](docs/source/examples/Einstein_Tensor_symbolic_calculation.ipynb)\n",
+    "\n",
+    "\n",
+    "## [Lambdify in Symbolic module](docs/source/examples/Lambdify_symbolic_calculation.ipynb)\n",
+    "\n",
+    "[![symbol](docs/source/examples/lambdifygenerated.png)](docs/source/examples/Lambdify_symbolic_calculation.ipynb)\n"
```
Comment (ritzvik) [inline index.ipynb]: you would need a new gallery image for the lambdify example

**Comment by ritzvik** [comment]
After this, please squash n rebase

**Comment by ritzvik** [comment]
Somehow, you messed up the commits 🤣 . No problem, let me take it from here.

**Comment by ritzvik** [comment]
> Somehow, you messed up the commits 🤣 . No problem, let me take it from here.

Okay, you corrected them!

**Comment by OVSofia** [comment]
> the

done xDD don't worry

**Comment by OVSofia** [comment]
> > Somehow, you messed up the commits rofl . No problem, let me take it from here.
> 
> Okay, you corrected them!

I'm still wrong with git, but at least I know how to fix it xD

**Comment by OVSofia** [comment]
> @OVSofia I did a few changes. Can you see the way I have committed, and how GitHub recognizes that ?

Yes, If you want I can change the last commit to add you as co-author

**Comment by ritzvik** [comment]
> > @OVSofia I did a few changes. Can you see the way I have committed, and how GitHub recognizes that ?
> 
> Yes, If you want I can change the last commit to add you as co-author

Chill! Leave it here. You did great! Anyways, I have to work really had now to find things new things for you to do. 😄

**Comment by ritzvik** [comment]
@OVSofia Maybe start working on the final report? Try making it on latex if you want, there is a learning curve but it would help you write research papers at some point in your career. Or do it in docs, wherever you like!

**Comment by OVSofia** [comment]
> @OVSofia Maybe start working on the final report? Try making it on latex if you want, there is a learning curve but it would help you write research papers at some point in your career. Or do it in docs, wherever you like!

I used latex for all my reports, so I am quite familiar with it. How long should be the final report?

**Comment by ritzvik** [comment]
> > @OVSofia Maybe start working on the final report? Try making it on latex if you want, there is a learning curve but it would help you write research papers at some point in your career. Or do it in docs, wherever you like!
> 
> I used latex for all my reports, so I am quite familiar with it. How long should be the final report?

I don't know. Whatever is suitable. Look at https://socis.esa.int/timeline/ and maybe you may get an idea.

**Pull request merged**